addOns/pscanrules/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## Unreleased
 ### Added
 - The X-Backend-Server Scan Rule now includes example alert functionality for documentation generation purposes (Issue 6119).
+- The Cache Control Scan Rule now includes example alert functionality for documentation generation purposes (Issue 6119).
```

Comment:
See https://github.com/zaproxy/zap-extensions/pull/4705#discussion_r1246668143

Reply:
The other PR was merged in the meantime, this should just need to add to the existing entry.